packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -280,6 +283,32 @@ class EditableText extends StatefulWidget {
   /// The text style to use for the editable text.
   final TextStyle style;
 
+  /// {@template flutter.widgets.editableText.strutStyle}
+  /// The strut style used for the vertical layout.
+  ///
+  /// [StrutStyle] is used to establish a predictable vertical layout.
+  /// Since fonts may vary depending on user input and due to font
+  /// fallback, [StrutStyle.forceStrutHeight] is enabled by default
+  /// to lock all lines to the height of the base [TextStyle], provided by
+  /// [style]. This ensures the typed text fits within the alotted space.
+  ///
+  /// If null, the strut used will is inherit values from the [style] and will
```

Comment:
If null, a [StrutStyle] based on [style], with [StrutStyle.forceStrutHeight] set to true, will be used.  If  [style] is null (the default) then the theme's [TextStyle] will be used instead.

However: we don't seem to be defaulting style to the theme's style?

Reply:
This behavior is actually inherited from the TextStyle taking the theme's style, which the StrutStyle then in turn inherits.